src/main/java/org/mitre/synthea/engine/Logic.java

```diff
@@ -353,6 +356,7 @@ public boolean test(Person person, long time) {
   private abstract static class ActiveLogic extends Logic {
     protected List<Code> codes;
     protected String referencedByAttribute;
+    protected String value_set;
```

Comment:
Field names in Java code should be camelCased, ie valueSet,, and the field names on the JSON should be snake_cased, ie value_set. It's a little bit "magical" but the engine converts one to the other when the JSON is parsed.